lib/json/jsonfile.js

```diff
@@ -2,15 +2,21 @@
 
 const jsonFile = require('jsonfile')
 
+function write () {
+  jsonFile.writeFile.apply({ spaces: 2 }, arguments)
```

Comment:
Seems to me it should follow the `jsonfile` default of `null` spaces. Don't you think?